Add modal themes

daniel-tabarcea opened the pull request:
- added the first theme, the main one with the a blue title bar
- connects unicef/etools-issues#412

daniel-tabarcea committed:
feature(add-modal-themes):
- added the first theme, the main one with the a blue title bar
- connects unicef/etools-issues#412

adi130987 commented on etools-dialog.html:
```diff
@@ -30,14 +30,15 @@
       }
       paper-dialog .dialog-title {
-        margin: 0 0 15px 0 !important;
+        margin: 0 !important;
```
avoid using `!important`

adi130987 commented on etools-dialog.html:
```diff
@@ -30,14 +30,15 @@
         min-height: 26px;
         @apply(--etools-dialog-title);
       }
       paper-dialog-scrollable {
```
Use the `--paper-dialog-scrollable` mixin applied to paper-dialog-scrollable and then add a new element specific mixin like `etools-dialog-scrollable` to be used in the app for more customizations.  Avoid using `!important`.

daniel-tabarcea committed:
feature(add-modal-themes):
 - added the confirmation modal theme

daniel-tabarcea committed:
feature(add-modal-themes):
- fixed an issue with a rogue close button
- update paper-dialog package version

daniel-tabarcea committed:
Fixed cancel button color on default theme

daniel-tabarcea committed:
feature(add-modal-themes):
- updated warning dialog theme

adi130987 commented on dynamic-dialog-behavior.html:
```diff
@@ -24,7 +24,7 @@
         dialog.dialogTitle = title;
       }
-
+      dialog.theme = 'confirmation';
```
`'confirmation'`  should be a `createDialog` parameter.

adi130987 commented on etools-dialog.html:
```diff
@@ -30,17 +30,22 @@
       }
       paper-dialog .dialog-title {
-        margin: 0 0 15px 0 !important;
+        margin: 0 !important;
```
`!important` not needed

daniel-tabarcea committed:
feature(add-modal-themes):
- refactored the code, removed second close button
- added a no-padding option

daniel-tabarcea committed:
feature(add-modal-themes):
- refactored css, changed from paragraph as content container to div

daniel-tabarcea committed:
feature(add-modal-themes):
- fixed issues caused by the shadow-dom

daniel-tabarcea committed:
css clean-up

daniel-tabarcea committed:
added parameter for use of behavior with no padding

daniel-tabarcea committed:
more css updates

daniel-tabarcea committed:
fixed miscommited code, updated readme

daniel-tabarcea committed:
Added a border above the buttons container
